### PR TITLE
dotCMS/core#16469 Fix and update test for key-value

### PR DESCRIPTION
**Pull request by fmontes**

### CODE REVIEW

File: projects/dotcms-field-elements/src/components/dot-key-value/key-value-form/key-value-form.e2e.ts [hunk 1]
```diff
@@ -212,6 +198,10 @@ describe('key-value-form', () => {
                 expect(spyAddEvent).not.toHaveReceivedEvent();
             });
         });
+
+        xdescribe('lostFocus', () => {
```
Comment (fmontes) [inline projects/dotcms-field-elements/src/components/dot-key-value/key-value-form/key-value-form.e2e.ts]: Can't test blur, the trick @freddyucv doesn't work.

Comment (alfredo-dotcms) [reply projects/dotcms-field-elements/src/components/dot-key-value/key-value-form/key-value-form.e2e.ts]: remove it then

File: projects/dotcms-field-elements/src/components/dot-key-value/dot-key-value.e2e.ts [hunk 2]
```diff
@@ -384,6 +393,19 @@ describe('dot-key-value', () => {
                 });
             });
         });
+
+        describe('statusChange', () => {
+            it('should emit on lost focus in autocomplete', async () => {
+                const form = await getForm();
+                form.triggerEvent('lostFocus', {});
```
Comment (alfredo-dotcms) [inline projects/dotcms-field-elements/src/components/dot-key-value/dot-key-value.e2e.ts]: TBH, not sure if this needs `await`

Comment (alfredo-dotcms) [reply projects/dotcms-field-elements/src/components/dot-key-value/dot-key-value.e2e.ts]: just checked not needed, so all test code that uses `triggerEvent` does not need it, and we need to remove it!

File: projects/dotcms-field-elements/src/components/dot-key-value/key-value-form/key-value-form.e2e.ts (hunk 1 above)
Comment (fmontes) [reply projects/dotcms-field-elements/src/components/dot-key-value/key-value-form/key-value-form.e2e.ts]: Need to find a way to test it

**Pull request merged**